### PR TITLE
Updated packages

### PR DESCRIPTION
**Pull request by wschoonveld**
Updated all packages except Cypress

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
+    <portal-target name="portal-target" />
   </div>
-
-  <portal-target name="portal-target" />
-</div>
 </template>
 
 <script>
-import CookieLaw from 'vue-cookie-law'
+// import CookieLaw from 'vue-cookie-law'
```
Comment (SiggyF) [inline src/App.vue:40]: remove? Why is this disabled?

File: src/App.vue [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
 import AppHeader from '../src/components/AppHeader.vue'
 
 import mapConfig from './map.config.js'
 
 export default {
   components: {
-    AppHeader,
-    CookieLaw
+    AppHeader//,
+    // CookieLaw
```
Comment (SiggyF) [inline src/App.vue:49]: Remove?

File: src/App.vue [hunk 3]
```diff
@@ -104,51 +104,51 @@ export default {
...
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  padding: 0;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
-  white-space: nowrap;
-  border: 0;
-}
+  @import './styles/rws.css';
```
Comment (SiggyF) [inline src/App.vue:107]: Check spacing / indenting

File: src/components/CombinePopup.vue [hunk 4]
```diff
@@ -88,8 +90,14 @@ export default {
...
     const scenarioInfo = await getScenarioInfo(scenarioIds)
     /* extract number of features and bands per layer */
-    this.$set(this, 'bandCounts', scenarioInfo.properties.bandCounts)
-    this.$set(this, 'featureCount', scenarioInfo.features.length)
+    // this.$set(this, 'bandCounts', scenarioInfo.properties.bandCounts)
+    // this.$set(this, 'featureCount', scenarioInfo.features.length)
+
+    this.bandCounts = scenarioInfo.properties.bandCounts
+    this.featureCount = scenarioInfo.features.length
+    // const reactiveObject = reactive(this)
```
Comment (SiggyF) [inline src/components/CombinePopup.vue:98]: remove

File: src/components/ExportCombinedPopUp.vue [hunk 5]
```diff
@@ -138,7 +138,7 @@ export default {
           if (result.export_url) {
             window.location = result.export_url
           }
-          this.$emit('close')
+          this.$emit('onClose')
```
Comment (SiggyF) [inline src/components/ExportCombinedPopUp.vue:141]: if close is the event, would onClose  be a callback?

File: src/components/FilterPopup.vue [hunk 6]
```diff
@@ -50,13 +50,29 @@
...
 import { probabilityConfig } from '@/lib/probability-filter'
-import store from '@/store'
 import PopUp from '@/components/PopUp'
 
+import store from '@/store'
+
 export default {
   components: { PopUp },
+  setup () {
+    const store = useStore()
```
Comment (SiggyF) [inline src/components/FilterPopup.vue:68]: overwrites store

File: src/components/LayerControl.vue [hunk 7]
```diff
@@ -1,105 +1,106 @@
...
+      <!-- TODO: this is not a layer setting. Move this to an application settings pane. -->
+    <div v-if="!breachId && layerOptions.length > 1" class="layer-control__options">
+      <!-- TODO: this now  shows up for each band reorganize -->
       <label>
         <layer-control-select
           name="layer-variant"
           :options="layerOptions"
-          v-model="selectedLayerIndex"
-          @change="selectLayerOption($event)"
+          v-model:value="selectedLayerIndex"
```
Comment (SiggyF) [inline src/components/LayerControl.vue:56]: why :value?

**Comment by sjoerdbeentjes** [comment]
closed since updating all dependencies was solved in another PR

**Pull request closed without merging**